framework/OpenMod.EntityFrameworkCore.MySql/OpenModMySqlDbContext.cs

```diff
@@ -74,23 +49,11 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
             var connectionStringAccessor = m_ServiceProvider.GetRequiredService<IConnectionStringAccessor>();
             var connectionString = connectionStringAccessor.GetConnectionString(connectionStringName);
 
-            optionsBuilder.UseMySQL(connectionString,
+            optionsBuilder.UseMySql(connectionString,
```

Comment:
This should be removed and done in a different way, e.g. in container/service configuration when you add the DbContext. DbContext itself should not care what the underlying provider is